<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TORNO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mov_x_l">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Movimiento en X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mov_y_l">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Movimiento en Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mov_x_value">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="mov_y_value">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="torno">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TORNO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tare">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Tare</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_x">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_y">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="mover_x">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mover X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mover_y">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mover Y</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSalir">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuSalir"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
